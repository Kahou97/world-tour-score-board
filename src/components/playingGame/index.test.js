import Component from "./index";
import React from "react";
import renderer from "react-test-renderer";
const MODULE = '[COMPONENTS]'

test(`${MODULE} gameList`, () => {
  const game = {
    homeTeam: "home",
    homeScore: 0,
    awayTeam: "away",
    awayScore: 0,
  };
  const comp = renderer
    .create(<Component game={game} onEnd={() => {}} />)
    .toJSON();
  expect(comp).toMatchSnapshot();
});
