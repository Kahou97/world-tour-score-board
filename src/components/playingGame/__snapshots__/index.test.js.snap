// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[COMPONENTS] gameList 1`] = `
<div
  className="general"
>
  home
   - 
  away
  : 
  0
   - 
  0
  <button
    onClick={[Function]}
  >
    End Game
  </button>
</div>
`;
